alpm/README.md

```diff
@@ -25,7 +25,7 @@ previous versions if the API was not changed.
 
 alpm.rs also supports the pacman git master via the git feature.
 
-Currently alpm.rs supports libalpm v13.x.x.
+Currently alpm.rs supports libalpm v15.x.x.
```

Comment:
Is there a reason that there are two (almost) identical README's? Should we delete the one in `alpm`?

Reply:
I believe it stems back from before crates.io supported symlinks. alpm and alpm-sys both use the same README. Should just be a symlink now.